tests/samples/test_api.py

```diff
@@ -188,33 +166,114 @@ async def test_find(
                 "pathoscope": False,
                 "all_read": True,
                 "labels": [label_3.id],
+                "workflows": {"aodp": "none", "nuvs": "none", "pathoscope": "none"},
             },
         ],
         session=None,
     )
+    return client
 
-    path = "/samples"
-    query = []
 
-    if find is not None:
-        query.append(f"find={find}")
+@pytest.mark.apitest
+class TestFindSamples:
+    @pytest.mark.parametrize("find", [None, "gv", "sp"])
+    async def test_term(
+        self,
+        find,
+        fake2,
+        spawn_client,
+        static_time,
+        snapshot,
+    ):
+        client = await setup_initial_test_setup(fake2, spawn_client, static_time)
+        path = "/samples"
+        query = []
+        if find is not None:
+            query.append(f"find={find}")
+        if query:
+            path += f"?{'&'.join(query)}"
```

Comment:
I suspect this can be greatly simplified as you know that there will only ever be zero or one `find` term passed.